lms/templates/lti_provider/user-auth-error.html

```diff
@@ -0,0 +1,24 @@
+<%page expression_filter="h" />
+<%namespace name='static' file='../static_content.html'/>
+<%!
+from django.utils.translation import gettext as _
+from openedx.core.djangolib.markup import HTML, Text
+%>
+<%inherit file="../main.html" />
+
+<section class="outside-app">
+  <h1>
+    ${Text(_("There was an error when loading this module!"))}
+  </h1>
+  <p>
+    ${Text(_("This module is available only for users who are signed into {platform}. "
+        "Kindly sign in and refresh this page to load the content."
+      )).format(
+        platform=HTML(u"<em>{}</em>").format(static.get_platform_name())
```

Comment:
```suggestion
        platform=HTML("<em>{}</em>").format(static.get_platform_name())
```

Nit: fortunately, we already use Unicode :)